src/pages/Home/Components/ProfileMenu.tsx

```diff
@@ -22,16 +38,38 @@ const ProfilMenuData = [
   { Icon: BusinessIcon, text: "Upgarde to Business" },
   { Icon: DownloadIcon, text: "Download apps" },
 ];
-const ProfileMenu = () => {
+const ProfileMenu: FC<CardProps> = ({ DisplayName, DisplayEmail }) => {
   const navigate = useNavigate();
   const [render, setRender] = useState(false);
 
+  const [name, setName] = useState(DisplayName);
+  const [email, setEmail] = useState(DisplayEmail);
+
   useEffect(() => {
     if (render) {
       navigate(PATHS.LOGIN);
     }
+    const userId = sessionStorage.getItem("userId");
+    Axios.get(`${API.getUser}/${userId}`)
+      .then((response) => {
+        if (response.data.user.userName) {
```

Comment:
no need to **if**. Because If the user does not have userName, won't see this page. Will see the onboard page. Remember?